R/migrate.R

```diff
@@ -249,14 +249,14 @@ archive_migrate_depends <- function(depends, parameters) {
   }
   ## Seen in rtm_incoming_serology/20200603-204022-70b8bfa5
   if (is.null(depends$index)) {
-    depends$index <- as.integer(factor(depends$id))
+    depends$index <- as.integer(factor(paste0(depends$id, depends$id_requested)))
```

Comment:
I don't *think* we should ever get this where they have the same ID, but have requested it in slightly different ways. But have put this in to be extra careful.

Reply:
one as latest, the other as a fixed id?

Reply:
(that said, I think this is only present in very old reports)